src/server/plugins/engine/components/NumberField.test.ts

```diff
@@ -199,8 +228,8 @@ describe('NumberField', () => {
         const viewModel = componentCustom.getViewModel(getFormData(99.99))
 
         expect(viewModel.attributes).toEqual(
-          expect.objectContaining({
-            step: '0.01'
+          expect.not.objectContaining({
+            inputmode: 'numeric'
           })
         )
```

Comment:
They're definitely a lot shorter

```suggestion
        expect(viewModel.attributes).not.toHaveProperty('inputmode')
```